### DISCUSSION TITLE
Lesson 11: NFTs not appearing at OpenSea-testnet webpage.

### DISCUSSION THREAD

**Discussion by mateuscgodoy** [smartcontractkit/full-blockchain-solidity-course-py#537] [Q&A]
Hello guys,

Just finished lesson 11 and for whatever reason everthing runs fine, but my NFT's are not appearing at the OpenSea webpage. I tried to refresh the metadata a couple of times and waited for more than a hour. No success so far tho.

Contract number:  0xd74A38D5740ddd5A008368129a4Ca6AF3E64d5A1

Got no idea what could be causing this issue, tried to delete de terminal, recompile brownie, and the whole contract again, but the problem seems to persist!

Thanks in advance!

![nft_collection](https://user-images.githubusercontent.com/57444793/145692750-5604bcc2-c454-4dc3-a71a-41ec749bd48b.png)

![nft_collection_2](https://user-images.githubusercontent.com/57444793/145692804-a19f191d-7889-4196-bb2a-50cd0e155a1c.png)

**Comment by cromewar** [comment]
Hello @thyiron2009 same here: 

![image](https://user-images.githubusercontent.com/20199180/145693948-1fe746e3-4cca-49eb-b48e-ed6f028c803d.png)

the img is hosted using ipfs, the node may probably be down.

@PatrickAlphaC did aything happen with the NFT's img on IPFS?

**Reply by PatrickAlphaC** [reply]
I think opensea is being a little confused... I can see the tokenURI correctly on your contract. This is likely an issue with opensea right now.

**Reply by cromewar** [reply]
Nice to know that, let's wait opensea fix this.
